controls/treeview/item-template.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Item Template
+page_title: .NET MAUI TreeView Documentation - Item Template
+description: Review how to customize the default text displayed in the Telerik .NET MAUI TreeView control by using the item template. 
+position: 7
+slug: treeview-item-template
+---
+
+# .NET MAUI TreeView Item Template
+
+Customize the text displayed in the TreeView item by using the `ItemTemplate` property. The `ItemTemplate` applies on TreeView level. The `ItemTemplate` does not include the checkbox, image and expand/collapse indicator. These elements are part of the `ControlTemplate`. For more details about how to customize the TreeView Control Template review the [Control Template]({%slug treeview-control-template%}) article. 
```

Comment:
Good! You already have the answers to the questions below:

1. What is an `ItemTemplate`?
2. What do I use it for?
3. When do I use it?
4. Why don't I use something else?

All you need now is to use the correct order and to be more precise when writing.

I am not sure what exactly "The `ItemTemplate` applies on TreeView level" means, so please correct me if I got this wrong.

```suggestion
The TreeView `ItemTemplate` property enables you to customize the text displayed in the TreeView items. Using this property allows you to apply text styles to all TreeView items.

In addition to the item's text, TreeView allows you to customize the checkbox, image, and expand/collapse indicator. For this purpose, use the [Control Template]({%slug treeview-control-template%}).
```